### ISSUE TITLE
Windows keyboard input doesn't work

### ISSUE THREAD

**Issue by olilarkin** [gskinnerTeam/flutter-folio#71]
I keep on getting excited about flutter, then trying it and immediately finding big issues

I downloaded the flutter-folio app from the windows store and when trying to type my email in the email field to create an account there are all sorts of strange behaviours with text input. None of my delete keys work. Text randomly gets capitalized. This is on a UK keyboard.

**Comment by SebTardif** [comment]
Right, did the same in March 3rd, 2022. Backspace doesn't work in the first field if Sign-up... That's what we call **fast fail**!
Thank you to make it clear it's not ready!

**Comment by rp1231** [comment]
I faced the same issues as well

**Comment by NeXT405** [comment]
@SebTardif 
On my Wondows 10 its not the backspace. If I try to write an @ with "Alt Gr + 2" it seems like the ctrl button keeps pressed? Even if I close the application. Have to press the "Ctrl" button again to "unstuck" it. 

Let me know if you need more informations or a log-file. 

PS: It's a a swiss german layout.
